connector/src/main/java/org/geysermc/connector/entity/PlayerEntity.java

```diff
@@ -298,6 +316,71 @@ public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession s
         }
     }
 
+    public void updateBoundingBox(Vector3f position) {
+        updateBoundingBox(Vector3d.from(position.getX(), position.getY(), position.getZ()));
+    }
+
+    public void updateBoundingBox(Vector3d position) {
+        if (boundingBox == null) {
+            System.out.println("BBnull");
+            boundingBox = new BoundingBox(position.getX(), position.getY(), position.getZ(), 0.60001, 1.8, 0.60001);
+        } else {
+            // TODO: Make bounding box smaller when sneaking
+            boundingBox.setMiddleX(position.getX());
+            boundingBox.setMiddleY(position.getY() + 0.9); // (EntityType.PLAYER.getOffset() / 2));
+            // System.out.println("Offset: " + (EntityType.PLAYER.getOffset() / 2));
+            boundingBox.setMiddleZ(position.getZ());
+        }
+    }
+
+    public static final double COLLISION_TOLERANCE = 0.000001;
```

Comment:
A Javadoc comment here would be nice.

Reply:
I might move this somewhere else or rename it, for now I'm just making it work

Reply:
Done